desktop/apps/consignments2/client/actions.js

```diff
@@ -14,12 +35,42 @@ export function createSubmission () {
   }
 }
 
+export function fetchArtistSuggestions (value) {
+  return async (dispatch, getState) => {
+    try {
+      request.get(`${sd.API_URL}/api/v1/match/artists`)
```

Comment:
I had some trouble trying to mock this request/response in a test. Was also wondering if `superagent` is our desired request library or if there's a different one I should be using?